reframe_config.py

```diff
@@ -432,18 +438,11 @@ def spack_root_to_path():
                         'upper=%(check_perf_upper_thres)s|'
                         'units=%(check_perf_unit)s|'
                         'spack_spec=%(check_spack_spec)s|'
-                        'variables=%(check_variables)s'
+                        'env_vars=%(check_env_vars)s'
```

Comment:
The `check_perf_*` fields above probably have to be modified as well, according to https://reframe-hpc.readthedocs.io/en/stable/config_reference.html#config.logging.handlers_perflog.format_perfvars . Or turn on compatibility with `logging.perflog_compat`

Reply:
Thanks, done.  Example of new perflog:
```
job_completion_time|version|info|jobid|num_tasks|num_cpus_per_task|num_tasks_per_node|flops_var=flops_value|flops_ref|flops_lower|flops_upper|flops_unit|spack_spec|env_vars
2023-03-08T17:38:06|reframe 4.1.0|SombreroBenchmark /cc391b86 @generic:default+default|jobid=74922|num_tasks=1|num_cpus_per_task=1|num_tasks_per_node=1|flops=1.15|ref=1|lower=None|upper=None|units=Gflops/seconds|spack_spec=sombrero@2021-08-16|env_vars={"OMP_NUM_THREADS": "1"}
2023-03-08T17:38:58|reframe 4.1.0|SombreroBenchmark /cc391b86 @generic:default+default|jobid=75084|num_tasks=1|num_cpus_per_task=1|num_tasks_per_node=1|flops=1.16|ref=1|lower=None|upper=None|units=Gflops/seconds|spack_spec=sombrero@2021-08-16|env_vars={"OMP_NUM_THREADS": "1"}
2023-03-08T17:40:41|reframe 4.1.0|SombreroBenchmark /cc391b86 @generic:default+default|jobid=75511|num_tasks=1|num_cpus_per_task=1|num_tasks_per_node=1|flops=1.12|ref=1|lower=None|upper=None|units=Gflops/seconds|spack_spec=sombrero@2021-08-16|env_vars={"OMP_NUM_THREADS": "1"}
```